### PR TITLE
[zh] sync/releases/1.8.x/announcing-1.8.3/index.md

### PR DESCRIPTION
**Pull request by liangyongzhenya**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ x ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+linktitle: 1.8.3
+subtitle: 补丁发布
+description: Istio 1.8.3 补丁发布。
+publishdate: 2021-02-08
+release: 1.8.3
+aliases:
+- /news/announcing-1.8.3
+---
+
+这个版本包含了错误修复，以提高稳定性。本发布说明了 Istio 1.8.2 和 Istio 1.8.3 之间的不同之处。
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:12]:
```suggestion
这个版本包含了错误修复，以提高稳定性。本发布说明描述了 Istio 1.8.2 和 Istio 1.8.3 之间的不同之处。
```

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+{{< relnote >}}
+
+## 安全
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:16]:
```suggestion
## 安全
```
Give an anchor please.

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+
+Istio 1.8.3 将不会包含之前在 [discuss.istio.io](https://discuss.istio.io/t/upcoming-istio-1-7-8-and-1-8-3-security-release/9593) 上公布的安全修复程序。
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:19]:
```suggestion
Istio 1.8.3 将不会包含之前在 [discuss.istio.io](https://discuss.istio.io/t/upcoming-istio-1-7-8-and-1-8-3-security-release/9593) 上公布的安全修复。
```

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+目前还没有计划的日期。请放心，这是 Istio 产品安全工作组的首要任务，但由于细节问题，我们无法发布更多信息，延迟的公告可以在 [这里](https://discuss.istio.io/t/istio-1-7-8-and-1-8-3-cve-fixes-delayed/9663) 查看。
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:20]:
```suggestion
目前还没有计划的日期。当然这是 Istio 产品安全工作组的首要任务，但由于细节问题，我们无法发布更多信息，有关此延迟的公告请参阅[这里](https://discuss.istio.io/t/istio-1-7-8-and-1-8-3-cve-fixes-delayed/9663)。
```

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## 变更
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:22]: need to add an anchor.

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+- **修复** Envoy 中 TLS 初始化期间聚合集群的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:24]:
```suggestion
- **修复** 修复了 Envoy 中 TLS 初始化期间聚合集群的问题。
```
参照本句修复以下的风格

**Review by windsonsea — COMMENTED** [review]
Please remember those comments a reviewer has provided to you before. Thanks for your contribution.

请谨记 reviewer 给的 comments，之所以会给你 negative 反馈，是因为你需要的进步的空间还很大。如果三番五次还是同样的粗心大意，招致批评是非常正常的。

File: content/zh/news/releases/1.8.x/announcing-1.8.3/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  ([Issue #28620](https://github.com/istio/istio/issues/28620))
+
+- **修复** 使用 `Sidecar` `ingress` 配置时，Istio 1.8 导致 1.7 代理配置错误的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.8.x/announcing-1.8.3/index.md:27]:
```suggestion
- **修复** 修复了使用 `Sidecar` `ingress` 配置时会导致 Istio 1.8 不正确地配置 Istio 1.7 代理的问题。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
Please change the style by following comments and the newest release notes.

**Comment by kebe7jun** [comment]
1.8.x 的版本过时了，不需要再维护了哈。

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by windsonsea** [comment]
This will be closed since the version is outdated. We need not to maintain it further.

**Pull request closed without merging**